t/io_uring.c

```diff
@@ -740,8 +740,8 @@ int main(int argc, char *argv[])
 		} else
 			rpc = ipc = -1;
 		file_depths(fdepths);
-		printf("IOPS=%lu, IOS/call=%ld/%ld, inflight=(%s)\n",
-				this_done - done, rpc, ipc, fdepths);
+		printf("IOPS=%lu, BWPS=%lu MB, IOS/call=%ld/%ld, inflight=(%s)\n",
+				this_done - done, (this_done - done) * bs / 1048576, rpc, ipc, fdepths);
```

Comment:
I'd change that so it reads:

BW=2915MiB/s

to make it really clear. Probably add a variable that does holds iops, ala:

iops = this_done - done;

and then use that for both IOPS and BW numbers.

Reply:
I found odd to have IOPS=x and BW=y/sec while PS meant /sec ;)
That's why I tried to get a similar report IOPS/BWPS.
Ack for MiB.

Reply:
IOPS is pretty commonly used and understood, BWPS is not ;-)

Reply:
thought to same but tried ;)
Fixed.